cypress/integration/smoke/index.js

```diff
@@ -23,6 +24,7 @@ import authorizationTests from './authorization'
 import profileSettingsTests from './profile-settings'
 
 export default [
+  ...checkCollapsingFields,
```

Comment:
I think putting this after the `applicationTests` is better, since these tests also depend on the application tests to pass.